subproviders/etherscan.js

```diff
@@ -88,13 +88,19 @@ function toQueryString(params) {
   }).join('&')
 }
 
-function etherscanXHR(proto, module, action, params, end) {
+function etherscanXHR(proto, get, module, action, params, end) {
   var uri = proto + '://api.etherscan.io/api?' + toQueryString({ module: module, action: action }) + '&' + toQueryString(params)
   // console.log('[etherscan] request: ', uri)
 
+  // URLs have a length in many browsers, switch to POST in that case
+  // http://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers
+  if (uri.length > 2000) {
```

Comment:
@kumavis this part was not taken into `master` and `eth_sendRawTransaction` still uses `GET`.